ci/test.yml

```diff
@@ -341,16 +346,16 @@ connect-popup legacy npm package:
       - TEST_FILE: ["popup-close.test"]
       - TEST_FILE: ["popup-error-page.test"]
 
-connect-popup-webextension:
-  extends: .e2e connect-popup-webextension
+connect-explorer-webextension:
```

Comment:
on line 223 this test is called `.e2e connect-popup:` I suggest unifying both under one name, either  based or popup or connect-explorer.

Reply:
maybe maybe, we should call it using package names we are actually end-to-end testing here? and those packages are @trezor/connect-web and @trezor/connect-webextension ?

Reply:
or maybe we want to have both
- e2e for popup
- e2e for connect/web (connect-explorer), connect/webextension (connect-explorer-webextension)

### e2e for popup
- basically only testing "methods". the goal here is to render everything popup can render. 
- useful for visual regression testing, screenshoting documentation

### e2e for packages
- all other tests expcept for `methods`. what kind of content popup renders doesnt matter, it is enough to test it once.
- here what is importand is to test how communication with popup/core/iframe is established, how it opens, closes itself etc.

Reply:
There will also be E2E tests for core-in-popup, so overall we will have at least:  
- `connect/web` in core-in-iframe mode
- `connect/web` in core-in-popup mode
- `connect/web` in webextension iframe (MV2)
- `connect/webextension`

I think we need to test methods for each of them, because the method of communication can also affect methods calls, as we saw with that race condition.

Reply:
ok, lets discuss here https://github.com/trezor/trezor-suite/issues/11228